### PR TITLE
Helm repo add updates for k8s docs

### PR DESCRIPTION
**Pull request by stevenGravy**
Use repo add to keep consistent across helm deployment docs.   This can also lead to issues since these two examples did not have the `helm repo update` which will prevent getting the latest teleport helm version.

### CODE REVIEW

**Review by tigrato — APPROVED** [review]

**Comment by ptgott** [comment]
We include the `helm repo add teleport` command in other pages as well—should we add this partial to all of them?

```bash
grep -Rl "\$ helm repo add teleport" docs/pages
docs/pages/access-controls/access-request-plugins/ssh-approval-mattermost.mdx
docs/pages/access-controls/access-request-plugins/ssh-approval-pagerduty.mdx
docs/pages/access-controls/access-request-plugins/ssh-approval-jira-server.mdx
docs/pages/access-controls/access-request-plugins/ssh-approval-jira-cloud.mdx
docs/pages/access-controls/access-request-plugins/ssh-approval-slack.mdx
docs/pages/deploy-a-cluster/helm-deployments/kubernetes-cluster.mdx
docs/pages/deploy-a-cluster/helm-deployments/digitalocean.mdx
docs/pages/kubernetes-access/getting-started.mdx
docs/pages/kubernetes-access/guides/multiple-clusters.mdx
docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx
docs/pages/management/guides/teleport-operator.mdx
docs/pages/management/guides/fluentd.mdx
docs/pages/try-out-teleport/local-kubernetes.mdx
docs/pages/installation.mdx
```

**Comment by stevenGravy** [comment]
> We include the `helm repo add teleport` command in other pages as well—should we add this partial to all of them?
> 
> 
> 
> ```bash
> 
> grep -Rl "\$ helm repo add teleport" docs/pages
> 
> docs/pages/access-controls/access-request-plugins/ssh-approval-mattermost.mdx
> 
> docs/pages/access-controls/access-request-plugins/ssh-approval-pagerduty.mdx
> 
> docs/pages/access-controls/access-request-plugins/ssh-approval-jira-server.mdx
> 
> docs/pages/access-controls/access-request-plugins/ssh-approval-jira-cloud.mdx
> 
> docs/pages/access-controls/access-request-plugins/ssh-approval-slack.mdx
> 
> docs/pages/deploy-a-cluster/helm-deployments/kubernetes-cluster.mdx
> 
> docs/pages/deploy-a-cluster/helm-deployments/digitalocean.mdx
> 
> docs/pages/kubernetes-access/getting-started.mdx
> 
> docs/pages/kubernetes-access/guides/multiple-clusters.mdx
> 
> docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx
> 
> docs/pages/management/guides/teleport-operator.mdx
> 
> docs/pages/management/guides/fluentd.mdx
> 
> docs/pages/try-out-teleport/local-kubernetes.mdx
> 
> docs/pages/installation.mdx
> 
> ```

Yes I'll check those

**Comment by stevenGravy** [comment]
done @ptgott

File: docs/pages/installation.mdx [hunk 1]
```diff
@@ -108,9 +108,8 @@ For instructions on running containers with these images, see
 We host our own Helm chart repository, which you can add with the following
```
Comment (ptgott) [inline docs/pages/installation.mdx:108]: We should delete this paragraph, since similar text is included in the partial

**Review by ptgott — APPROVED** [review]
Approved with some comments

**Pull request merged**